packages/@aws-cdk/aws-rds/README.md

```diff
@@ -74,6 +74,7 @@ const instance = new rds.DatabaseInstance(this, 'Instance', {
   instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE3, ec2.InstanceSize.SMALL),
   credentials: rds.Credentials.fromGeneratedSecret('syscdk'), // Optional - will default to 'admin' username and generated password
   vpc,
+  copyTagsToSnapshot: true, // Optional - will default to 'false'
```

Comment:
1. This is the wrong place for this comment - you added it to `DatabaseCluster`, while the example is for `DatabaseInstance`.
2. Maybe also explain what this does?

```suggestion
  copyTagsToSnapshot: true, // whether to save the cluster tags when creating the snapshot. Default is 'false'
```